index.html

```diff
@@ -3597,5 +3597,32 @@ <h3 id="applying">
         specification via the HTML5Apps project.
       </p>
     </section>
+    <section id="developer-guidance"></section>
+    <section class="appendix">
+      <h2>
+        Developer Guidance
+      </h2>
+      <p>
+        This section provides non-normative guidance for developers of web app manifest files 
+      </p>
+      <h3>
+        Additional Examples
+      </h3>
+      <p>
+        TODO: Add additional examples
+      </p>
+      <h3>
+        Interaction With Web Crawlers
+      </h3>
+      <p>
+        Like other web resources, a web app manifest should be accessible to any web browser or web crawler.
+      </p>
+      <p>
+        If a web app developer wants to block traffic from web crawlers, 
```

Comment:
Thanks @alrra.  I've replaced this PR with #499 and incorporated your suggestion of "block traffic" to "inform"